docs/organizer-guide/meetups/livestreaming-meetups.rst

```diff
@@ -0,0 +1,40 @@
+Livestreaming or Recording a Meetup
+===================================
+
+Whether due to health or environmental reasons or because you live somewhere where people are spread out over a large area, maybe you want to try live streaming a meetup.
+
+We welcome streams or uploads to the Podcast YouTube channel, but first, we have to add you to the channel. A few days ahead of time, message @chrisward @tomjohnson or @jaredmorgs or the #podcast Slack channel with your email address and we will add you to the channel.
```

Comment:
Do you have video capabilities? If not, maybe we should broaden this to include the WTD YouTube channel. I imagine most presenters would prefer to stream or record video/slides and not just audio.

Reply:
Yes, I think we covered this in a Slack chat recently? But this sat for a while of course :)

but i'll change the channel name here.